docs/storage_controller.md

```diff
@@ -101,7 +101,7 @@ changes such as a pageserver node becoming unavailable, or the tenant's shard co
 postgres clients to handle such changes, the storage controller calls an API hook when a tenant's pageserver
 location changes.
 
-The hook is configured using the storage controller's `--compute-hook-url` CLI option. If the hook requires
+The hook is configured using the storage controller's `--control-plane-url` CLI option. If the hook requires
```

Comment:
This should specify what the sub-paths of `control-plane-url` are -- otherwise the reader would have to read the code to find out that their notification hook needs to be called `notify-attach`

Reply:
added docs.